### DISCUSSION TITLE
How to run multiple database migration(3 class library) from single console application(.net core)?

### DISCUSSION THREAD

**Discussion by shailesh1984** [fluentmigrator/fluentmigrator#1643] [Q&A]
Hi Team,

Requirement : 
I have 3 database and 3 new class library and 1 console application.
Executing console application with these 3 migration DB's(class library).

DB : SQL Server 2018
Application : .NET Core
FluentMigration Version : 3.3.2

Please kindly help me and provide some migration code steps.

Thanks,
Shailesh M

**Comment by jzabroski** [comment]
Hi @shailesh1984 

I don't really understand this request.  Are you asking how to pass in multiple migration assemblies to the FluentMigrator.DotNet.Cli tool?
